packages/config-array/src/config-array.js

```diff
@@ -598,6 +581,13 @@ export class ConfigArray extends Array {
 		} else {
 			this.push(configs);
 		}
+
+		// On Windows, `path.relative()` returns an absolute path when given two paths on different drives.
+		// The namespaced base path is useful to make sure that calculated relative paths are always relative.
+		// On Unix, it is identical to the base path.
+		this.namespacedBasePath = path.toNamespacedPath(
```

Comment:
I was trying to avoid using anything more complicated than `path.relative()` and `path.resolve()`, because I'm not sure if there's a browser polyfill for other methods. Can you verify that `toNamespacedPath()` has an appropriate polyfill?

Reply:
Good point! I looked at `rollup-plugin-node-polyfills`, the polyfills we use in eslint/eslint, and there is no implementation for `toNamespacedPath` in the [`path`](https://github.com/ionic-team/rollup-plugin-node-polyfills/blob/master/polyfills/path.js) module polyfill, so I added a fallback to return the argument as is when `path.toNamespacedPath` is not defined. This is the same as the implementation on Posix, so basically on every non-Windows system ([source](https://github.com/nodejs/node/blob/92a25abca9dd23a723008d88b1ccb63ff40ec9b7/lib/path.js#L1293-L1300)).